README.md

```diff
@@ -17,7 +17,7 @@ Augur is now releasing a dramatically improved new version. It is also available
 - The `release` branch is a stable version of our new architecture, which features:
   - Dramatic improvement in the speed of large scale data collection (100,000+ repos). All data is obtained for 100k+ repos within 2 weeks.
   - A new job management architecture that uses Celery and Redis to manage queues, and enables users to run a Flower job monitoring dashboard.
-  - Materialized views to increase the snappiness of API’s and Frontends on large scale data.
+  - Materialized views to increase the snappiness of ```APIs``` and Frontends on large scale data.
```

Comment:
did you mean to surround this with backticks? looks like the intent was maybe just to remove the quote?